lerna.json

```diff
@@ -2,5 +2,5 @@
   "packages": ["packages/*", "plugins/*"],
   "npmClient": "yarn",
   "useWorkspaces": true,
-  "version": "0.1.1-alpha.18"
+  "version": "0.1.1-alpha.20"
```

Comment:
The lerna version doesn't seem to be updated when you do a yarn release. So after the previous alpha.19 release the version here was still pointing to alpha.18 and I think this is the version that gets picked up then you run create-app or create-plugin in the cli.

So updated this manually to match the new version.

Reply:
Oh interesting. That is indeed weird.